client/me/help/help-contact/index.jsx

```diff
@@ -620,6 +624,26 @@ const HelpContact = React.createClass( {
 
 		const supportVariation = this.getSupportVariation();
 
+		if ( supportVariation === SUPPORT_DIRECTLY && this.props.hasAskedADirectlyQuestion ) {
+			// We're taking the Directly confirmation outside the standard `confirmation` state object
+			// that other variations use, because we need this to persist even if the component is
+			// removed and re-mounted. Using `confirmation` in component state would mean you could
+			// ask a new Directy question every time you left the help section and came back.
+			const directlyConfirmation = {
+				title: this.props.translate( 'We\'re on it!' ),
```

Comment:
I think the linter would allow the following so we don't have to use escaping:
```
`We're on it!`
```
or 
```
"We're on it"
```